.travis.yml

```diff
@@ -31,11 +32,18 @@ jdk:
     - openjdk13
 
 before_install:
+    # upgrade shellcheck (https://github.com/koalaman/shellcheck#installing-a-pre-compiled-binary)
+    - wget -qO- "https://storage.googleapis.com/shellcheck/shellcheck-${SHELLCHECK_VERSION?}.linux.x86_64.tar.xz" | tar -xJv
+    - cp "shellcheck-${SHELLCHECK_VERSION}/shellcheck" "${BIN_DIR}"
+    - shellcheck --version
+    # run shellcheck
+    - ./scripts/shellcheck-build.sh
```

Comment:
This script is directly related to the build process. I would move it to the build directory